app/org/sagebionetworks/bridge/play/controllers/StudyController.java

```diff
@@ -130,6 +136,10 @@ public Result createStudy() throws Exception {
     }
 
     public Result createSynapse(String synapseUserId) throws Exception {
+        if (synapseUserId == null) {
+            return badRequest(Json.toJson("Synapse User ID cannot be null."));
+        }
```

Comment:
The pattern we usually use is throw new BadRequestException().

We also want to check for empty and blank strings. I recommend using StringUtils.isBlank().

The Controller is for Play-specific logic. We should move user-input validation to the Service.

Lastly, all code that we write should have corresponding unit tests.